integrations/cohere/LICENSE.txt

```diff
@@ -0,0 +1 @@
+404: Not Found
```

Comment:
That's a `curl` pipe that it didn't go according to plans I guess? 😅

Reply:
WTF